features_and_tools/cql.rst

```diff
@@ -0,0 +1,400 @@
+                        "data": "dXNpbmcgU3lzdGVtOw0KdXNpbmcgU3lzdGVtLkxpbnE7DQp1c2luZyBTeXN0ZW0uQ29sbGVjdGlvbnMuR2VuZXJpYzsNCnVzaW5nIEhsNy5DcWwuUnVudGltZTsNCnVzaW5nIEhsNy5DcWwuUHJpbWl0aXZlczsNCnVzaW5nIEhsNy5DcWwuQWJzdHJhY3Rpb25zOw0KdXNpbmcgSGw3LkNxbC5WYWx1ZVNldHM7DQp1c2luZyBIbDcuQ3FsLklzbzg2MDE7DQp1c2luZyBIbDcuRmhpci5Nb2RlbDsNCnVzaW5nIFJhbmdlID0gSGw3LkZoaXIuTW9kZWwuUmFuZ2U7DQp1c2luZyBUYXNrID0gSGw3LkZoaXIuTW9kZWwuVGFzazsNCltTeXN0ZW0uQ29kZURvbS5Db21waWxlci5HZW5lcmF0ZWRDb2RlKCIuTkVUIENvZGUgR2VuZXJhdGlvbiIsICIxLjAuMC4wIildDQpbQ3FsTGlicmFyeSgiQWdlRkhJUjQiLCAiMC4wLjEiKV0NCnB1YmxpYyBjbGFzcyBBZ2VGSElSNF8wXzBfMQ0Kew0KDQoNCiAgICBpbnRlcm5hbCBDcWxDb250ZXh0IGNvbnRleHQ7DQoNCiAgICAjcmVnaW9uIENhY2hlZCB2YWx1ZXMNCg0KICAgIGludGVybmFsIExhenk8UGF0aWVudD4gX19QYXRpZW50Ow0KICAgIGludGVybmFsIExhenk8Ym9vbD8+IF9faXMxOE9yQWJvdmU7DQoNCiAgICAjZW5kcmVnaW9uDQogICAgcHVibGljIEFnZUZISVI0XzBfMF8xKENxbENvbnRleHQgY29udGV4dCkNCiAgICB7DQogICAgICAgIHRoaXMuY29udGV4dCA9IGNvbnRleHQgPz8gdGhyb3cgbmV3IEFyZ3VtZW50TnVsbEV4Y2VwdGlvbigiY29udGV4dCIpOw0KDQoNCiAgICAgICAgX19QYXRpZW50ID0gbmV3IExhenk8UGF0aWVudD4odGhpcy5QYXRpZW50X1ZhbHVlKTsNCiAgICAgICAgX19pczE4T3JBYm92ZSA9IG5ldyBMYXp5PGJvb2w/Pih0aGlzLmlzMThPckFib3ZlX1ZhbHVlKTsNCiAgICB9DQogICAgI3JlZ2lvbiBEZXBlbmRlbmNpZXMNCg0KDQogICAgI2VuZHJlZ2lvbg0KDQoJcHJpdmF0ZSBQYXRpZW50IFBhdGllbnRfVmFsdWUoKQ0KCXsNCgkJdmFyIGFfID0gY29udGV4dC5PcGVyYXRvcnMuUmV0cmlldmVCeVZhbHVlU2V0PFBhdGllbnQ+KG51bGwsIG51bGwpOw0KCQl2YXIgYl8gPSBjb250ZXh0Lk9wZXJhdG9ycy5TaW5nbGVPck51bGw8UGF0aWVudD4oYV8pOw0KDQoJCXJldHVybiBiXzsNCgl9DQoNCiAgICBbQ3FsRGVjbGFyYXRpb24oIlBhdGllbnQiKV0NCglwdWJsaWMgUGF0aWVudCBQYXRpZW50KCkgPT4gDQoJCV9fUGF0aWVudC5WYWx1ZTsNCg0KCXByaXZhdGUgYm9vbD8gaXMxOE9yQWJvdmVfVmFsdWUoKQ0KCXsNCgkJdmFyIGFfID0gdGhpcy5QYXRpZW50KCk7DQoJCXZhciBiXyA9IGNvbnRleHQuT3BlcmF0b3JzLkNvbnZlcnQ8Q3FsRGF0ZT4oYV8/LkJpcnRoRGF0ZUVsZW1lbnQ/LlZhbHVlKTsNCgkJdmFyIGNfID0gY29udGV4dC5PcGVyYXRvcnMuQ2FsY3VsYXRlQWdlKGJfLCAieWVhciIpOw0KCQl2YXIgZF8gPSBjb250ZXh0Lk9wZXJhdG9ycy5HcmVhdGVyT3JFcXVhbChjXywgKGludD8pMTgpOw0KDQoJCXJldHVybiBkXzsNCgl9DQoNCiAgICBbQ3FsRGVjbGFyYXRpb24oImlzMThPckFib3ZlIildDQoJcHVibGljIGJvb2w/IGlzMThPckFib3ZlKCkgPT4gDQoJCV9faXMxOE9yQWJvdmUuVmFsdWU7DQoNCn0="
+                        }
+                    ]
+                }
+
+Firely Server CQL Plugin 
+------------------------
+
+This chapter details setting up Firely Server CQL Plugin and provides you with an example to get some hands-on experience with CQL library with Postman tool. 
+ 
+* Getting Started 
+* CQL Plugin Configuration 
+* Using Postman 
+
+Getting Started
+^^^^^^^^^^^^^^^^
+
+* Install Firely Server using `Basic installation — Firely Server documentation <https://docs.fire.ly/projects/Firely-Server/en/latest/getting_started/basic_installation.html>`_
+
+    .. warning::
+        Make sure that the license contains CQL Plugin. 
+
+CQL Plugin Configuration in Firely Server 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Edit appsettings.instance.json:
+
+    1. Set Repository to ``SQL`` and add the database configuration 
```

Comment:
Is it correct that CQL is currently only supported in combination with SQL? Than it is good to make a note/warning of that under the Firely Server CQL Plugin header

Reply:
@paul-l-fire can you confirm if this is true?